core/iwasm/fast-jit/fe/jit_emit_control.c

```diff
@@ -550,8 +550,12 @@ handle_op_else(JitCompContext *cc, uint8 **p_frame_ip)
         /* Clear value stack, restore param values and
            start to translate the else branch. */
         jit_value_stack_destroy(&block->value_stack);
+        /* Reload block parameters */
+        if (!load_block_params(cc, block)) {
+            return false;
+        }
```

Comment:
Why moving these lines here? Here we load block params again for else branch as similar to if branch, moving here causes loading again for if branch and missing for else branch?